seriously/SeriouslyCommands.py

```diff
@@ -728,7 +728,7 @@ def dupe_each_n_fn(srs):
     tmp = []
     while srs.stack:
         b = srs.pop()
-        tmp = [b]*a + tmp
+        tmp = [copy.deepcopy(b) for _ in range(a)] + tmp
```

Comment:
This wasn't necessary - `copy` in the global scope is a wrapper around `copy.copy` (well, now, `copy.deepcopy`) that properly handles generators.